lib/utils/pagination/paginationHelper.js

```diff
@@ -0,0 +1,11 @@
+const { LIMIT_HELPER, OFFSET_HELPER } = require('../../../constants');
+
+const pagination = (query) => {
```

Comment:
Function names are best when it is a verb.

Reply:
changed to paginate